pkg/kubelet/prober/prober_manager.go

```diff
@@ -299,6 +301,11 @@ func (m *manager) UpdatePodStatus(podUID types.UID, podStatus *v1.PodStatus) {
 			ready = true
 		}
 		podStatus.InitContainerStatuses[i].Ready = ready
+		podStatus.InitContainerStatuses[i].Started = &initContainerStartedValue
+	}
+	// enforcing ephemeral container statuses
+	for i := range podStatus.EphemeralContainerStatuses {
+		podStatus.EphemeralContainerStatuses[i].Started = &ephemeralContainerStartedValue
```

Comment:
is it true though? I think we need to check whether it's running. Maybe the same time we set the `StartedAt`. Or even later - `StartedAt` indicates we attempted to start it. `Started` is about it being in running state.

Reply:
`StartedAt` is set to the value reported by the container runtime, it happens in `kubelet_pods.go` when we initialise `Started` at `false`.
Maybe we should document somewhere that `Started` only means something in the context of prober_manager... which doesn't apply to ephemeral containers since these have no probes.